orchagent/main.cpp

```diff
@@ -143,14 +160,12 @@ int main(int argc, char **argv)
     int opt;
     sai_status_t status;
 
-    bool disableRecord = false;
-
     while ((opt = getopt(argc, argv, "m:hR")) != -1)
```

Comment:
i will leave this feature to a separate commit, let sairedis also take attribute so that the sairedis record files will be saved at the same directory.